### PR TITLE
[IMP] base: Honduras vat label and currency improvements

### PR DESCRIPTION
**Pull request by jose-pcg**
**Description of the issue/feature this PR addresses:**

- Adds address format and vat label to Honduras res.country data
- Improve HNL currency setup

**Current behavior before PR:**

- Missing address format and vat label
- Wrong HNL currency position and unit label

**Desired behavior after PR is merged:**

- Honduras address format and vat label comes as part of Honduras country data
- Have a correct HNL currency setup

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by d2isolutions** [comment]
Hi could you please help us with accepting this pull request.

**Comment by d2isolutions** [comment]
@jco-odoo Could you please help us with the review of this PR?

File: odoo/addons/base/data/res_country_data.xml [hunk 1]
```diff
@@ -698,6 +698,8 @@
             <field file="base/static/img/country_flags/hn.png" name="image" type="base64" />
             <field name="currency_id" ref="HNL" />
             <field eval="504" name="phone_code" />
+            <field eval="'%(street)s %(street2)s \n%(city)s %(state_name)s \n%(country_name)s'" name="address_format"/>
```
Comment (jco-odoo) [inline odoo/addons/base/data/res_country_data.xml:701]: Could you tell me what is different here than the standard?

**Comment by jco-odoo** [comment]
@oco-odoo Could you check this further?

File: odoo/addons/base/data/res_country_data.xml (hunk 1 above)
Comment (jlopezg) [reply odoo/addons/base/data/res_country_data.xml:701]: Got your point. Actually is the same as the standard format. 
I'm gonna remove that.

**Review by oco-odoo — COMMENTED** [review]
I'm okay with the diff. Just squash together the two commits touching the address line, and don't forget to put in every commit's message that it concerns Honduras (it's currently only done in the first one).

**Comment by jlopezg** [comment]
@oco-odoo done! Please check it out again.
Cc: @d2isolutions

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

**Comment by oco-odoo** [comment]
@jlopezg Done! Thanks for contributing :)

**Comment by oco-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**